training/tools/train_agent_utils.py

```diff
@@ -101,45 +99,36 @@ def initialize_hyperparameters(
     :param config_name: name of the hyperparameter file in /configs/hyperparameters
     :param n_envs: number of envs
     """
-    # when building new agent
-    if load_target is None:
-        hyperparams = init_params_new_agent(PATHS, config_name)
-    else:
-        hyperparams = init_params_loaded_agent(PATHS)
-
     import rosnav.model.custom_policy
     import rosnav.model.custom_sb3_policy
 
     # dynamically adapt n_steps according to batch size and n envs
     # then update .json
-    check_batch_size(n_envs, hyperparams["batch_size"], hyperparams["m_batch_size"])
-    hyperparams["n_steps"] = int(hyperparams["batch_size"] / n_envs)
+    check_batch_size(
+        n_envs,
+        config["rl_agent"]["ppo"]["batch_size"],
+        config["rl_agent"]["ppo"]["m_batch_size"],
+    )
+    config["rl_agent"]["ppo"]["n_steps"] = int(
+        config["rl_agent"]["ppo"]["batch_size"] / n_envs
+    )
```

Comment:
entpack doch die configs in eine neue variable dann musst du nicht so viel schreiben und ist übersichtlicher:

`params = config["rl_agent"]["ppo"]`